plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/Fabric8MultiTenantModule.java

```diff
@@ -29,7 +29,6 @@
   protected void configure() {
     LOGGER.info("Configuring {}", this.getClass().getName());
     bind(OpenShiftClientFactory.class).to(Fabric8OpenShiftClientFactory.class);
-    bind(KubernetesClientFactory.class).to(Fabric8OpenShiftClientFactory.class);
```

Comment:
I believe we can not remove this binding

Reply:
Added here https://github.com/eclipse/che/pull/17457/files#diff-e1932b1dcac1cd050ac5e68ca54d482fR118

Reply:
yeah, but the problem is that we need to use Fabric8OpenShiftClientFactory for Hosted Che - https://github.com/redhat-developer/rh-che/blob/master/plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/Fabric8OpenShiftClientFactory.java#L38